src/main/java/edu/harvard/iq/dataverse/marketplace/controller/ErrorResponseController.java

```diff
@@ -0,0 +1,58 @@
+package edu.harvard.iq.dataverse.marketplace.controller;
```

Comment:
Why not "org.dataverse.marketplace" instead? Short, sweet. More community-friendly.